src/index.css

```diff
@@ -1,17 +1,17 @@
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 div root{
-  font-size: 13px;  
+  font-size: 13px;
 }
```

Comment:
I don't think that this works either sadly. I suspect that this is meant to be either

```css
:root {
    font-size: 13px;
}
```
or 
```css
html {
    font-size: 13px;
}
```

would you mind also fixing this while you are here?